ui/src/app/applications/components/application-parameters/application-parameters.tsx

```diff
@@ -765,6 +765,10 @@ function gatherDetails(
     } else if (repoDetails.type === 'Helm' && repoDetails.helm) {
         const isValuesObject = source?.helm?.valuesObject;
         const helmValues = isValuesObject ? jsYaml.dump(source.helm.valuesObject) : source?.helm?.values;
+        console.log("isValuesObject:", isValuesObject);
```

Comment:
```suggestion
        console.log("isValuesObject:", isValuesObject);
```
are you trying to debug something? I don't think so we need this to be added.